charts/actions-runner-controller/templates/githubwebhook.deployment.yaml

```diff
@@ -94,17 +94,14 @@ spec:
               key: github_app_private_key
               name: {{ include "actions-runner-controller.githubWebhookServerSecretName" . }}
               optional: true
-        {{- if .Values.authSecret.github_basicauth_username  }}
         - name: GITHUB_BASICAUTH_USERNAME
           value: {{ .Values.authSecret.github_basicauth_username }}
-        {{- end }}
```

Comment:
I think this conditional block for `.Values.authSecret.github_basicauth_username` still has some value for consistency with others like `githubUploadURL`, `githubURL `, etc.
Let me revert that part only and merge other changes as-is 😄